Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:15.14-alpine AS build
+WORKDIR /src
+COPY . .
+RUN npm ci
+RUN npm run build
+
+FROM node:15.14-alpine as final
+RUN yarn global add serve
+WORKDIR /app
+COPY --from=build /src/build .
+EXPOSE 5000
```

Comment:
I can't see anywhere it would run on different port than the default `3000`, or am I wrong?

Reply:
It's exposed on 5000